.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
     # Create .splunkrc file with default credentials
     - echo host=localhost >> $HOME/.splunkrc
     - echo username=admin >> $HOME/.splunkrc
-    - echo password=changeme >> $HOME/.splunkrc
+    - echo password=changed! >> $HOME/.splunkrc
     # Set SPLUNK_HOME
     - export SPLUNK_HOME="/opt/splunk"
     # Pull docker image
```

Comment:
Can we remove the .travis.yml file actually? Feel free to do that in this PR

Reply:
Removed .travis.yml flie.